ciri/api/tests/test_api.c

```diff
@@ -0,0 +1,150 @@
+/*
+ * Copyright (c) 2018 IOTA Stiftung
+ * https://github.com/iotaledger/entangled
+ *
+ * Refer to the LICENSE file for licensing information
+ */
+
+#include <unity/unity.h>
+
+#include "ciri/api/api.h"
+#include "ciri/api/tests/defs.h"
+#include "consensus/test_utils/bundle.h"
+#include "consensus/test_utils/tangle.h"
+
+static char *test_db_path = "ciri/api/tests/test.db";
+static char *ciri_db_path = "ciri/api/tests/ciri.db";
+static connection_config_t config;
+static iota_api_t api;
+static tangle_t tangle;
+
+void setUp(void) {
```

Comment:
you're missing a test for the max_get_trytes limit?

Reply:
No it's there